chroma-manager/tests/integration/installation_and_upgrade/test_update_with_yum.py

```diff
@@ -64,12 +60,13 @@ def test_obsolete_chroma_diagnostics(self):
         """Test that chroma-diagnostics has been obsoleted"""
         servers = config['lustre_servers']
         addresses = [server['address'] for server in servers]
-        logger.debug('obsolete test: addresses %s' % (addresses))
         
         for address in addresses:
-            result = self.remote_command(address, 'chroma-diagnostics')
-            logger.debug('obsolete test: result of chroma-diagnostics on %s: %s' % (address, result.stdout))
-            self.assertEqual(result.stdout, "chroma-diagnostics no longer exists. Please use 'iml-diagnostics' instead.")
+            chroma_diagnostics_result = self.remote_command(address, 'chroma-diagnostics')
+            self.assertEqual(chroma_diagnostics_result.stdout.split('\n')[0], "chroma-diagnostics no longer exists. Please use 'iml-diagnostics' instead.")
+            
+            yum_installed_result = self.remote_command(address, 'yum list installed | grep chroma-diagnostics')
```

Comment:
`rpm -q chroma-diagnostics` would be much more precise rather than listing everything and searching that list (with `grep`).  `rpm -q` will return 0 if found, !0 if not found so you can test `yum_installed_result.rc` rather than looking for string output in `yum_installed_result.stdout`,